documentation/pom.xml

```diff
@@ -22,202 +22,219 @@
     <description>The Hibernate Search reference manual</description>
 
     <properties>
+        <asciidoctor.base-outputdir>${basedir}/target/asciidoctor/en-US</asciidoctor.base-outputdir>
```

Comment:
Yeah, it's mostly because I don't want to change the hierarchy on the website and the stable URL of the documentations.

I fixed the dir issue (and a small other one I didn't notice before) and force pushed an update.